internal/config/read.go

```diff
@@ -53,6 +53,7 @@ func (c *Config) readFromEnv(env env.Env) error {
 	c.TestFilePattern = env.Get("BUILDKITE_TEST_ENGINE_TEST_FILE_PATTERN")
 	c.TestFileExcludePattern = env.Get("BUILDKITE_TEST_ENGINE_TEST_FILE_EXCLUDE_PATTERN")
 	c.TestRunner = env.Get("BUILDKITE_TEST_ENGINE_TEST_RUNNER")
+	c.RetryForMutedTest = strings.ToLower(env.Get("BUILDKITE_TEST_ENGINE_DISABLE_RETRY_FOR_MUTED_TEST")) != "true"
```

Comment:
How come you need the `!=` logic here instead of `= "false"`?

Reply:
If I use `== "false"` check, I will essentially force user to set: `BUILDKITE_TEST_ENGINE_DISABLE_RETRY_FOR_MUTED_TEST`  to be `false` in order to use this feature.

Then it turns a opt-out feature flag, into a opt-in feature flag 😅